tools/barista/src/main.ts

```diff
@@ -64,9 +82,168 @@ async function buildPages(): Promise<void[]> {
   return Promise.all(files);
 }
 
+function getOverviewSectionItem(
+  filecontent: any,
+  category: string,
```

Comment:
Maybe rename this to section or navGroup? "category" could be confused with our content categories (components, patterns, resources...).